### ISSUE TITLE
Can I save camera input to video files?

### ISSUE THREAD

**Issue by gbanana** [Sascha-L/WPF-MediaKit#76]
Can I save camera input to video files?

**Comment by xmedeko** [comment]
No, it's not possible easily. You can save a screenshot, see the [Wiki Basic Examples](https://github.com/Sascha-L/WPF-MediaKit/wiki/Basic-Examples). So you can save a sequence of JPEG images, aka MPEG video (not optimal format, though). E.g. see this [SO thread](https://stackoverflow.com/questions/33753912).

Maybe you can go to the DirectX level to make better solution, but I am weak in the DirectX stuff, so cannot make you better advice, just use Google. E.g. see [File Writer Filter](https://msdn.microsoft.com/en-us/library/windows/desktop/dd375644(v=vs.85).aspx), or [this blog](http://www.pcpret.nl/index.php?c1=d1&r1=302&l1=31p) and you find others.

**Comment by jluisestrada** [comment]
Yes, it is possible if you modify the code (specially the directshow graph) and insert the GMF Filter http://www.gdcl.co.uk/gmfbridge/ so you can save to a file in parallel without skipping frames. I should have a sample somewhere. Let me upload it
